data_handler/handlers/order_books/uniswap_v2/main.py

```diff
@@ -94,3 +112,4 @@ def tick_to_price(self, tick: Decimal) -> Decimal:
     order_book = UniswapV2OrderBook(token_0, token_1)
     order_book.fetch_price_and_liquidity()
     print(order_book.get_order_book(), "\n")
+    token_amount = order_book.calculate_token_amount_price_change(Decimal("3750"), Decimal(0.05))
```

Comment:
The current price should be derived from the orderbook. It's the mid price between the best bid and ask.